includes/Generator/Term.php

```diff
@@ -38,7 +38,9 @@ public static function generate( $save = true, string $taxonomy = 'product_cat',
 
 		parent::maybe_initialize_generators();
 
-		if ( $taxonomy_obj->hierarchical ) {
+		if ( $taxonomy_obj->hierarchical && 'product_brand' === $taxonomy ) {
+            $term_name = ucwords( self::$faker->department( 1 ) );
```

Comment:
```suggestion
			$term_name = ucwords( self::$faker->deviceManufacturer() );
```
`department` was adding Commerce departments.